### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by ry**
- fix node_resolver
- fix warning

### CODE REVIEW

File: cli/tools/lint.rs [hunk 1]
```diff
@@ -550,7 +550,7 @@ pub(crate) fn get_configured_rules(
   );
 
   if configured_rules.is_empty() {
-    anyhow!("No rules have been configured");
+    return Err(anyhow!("No rules have been configured"));
```
Comment (dsherret) [inline cli/tools/lint.rs:553]: Interesting that this wasn't a clippy failure or something.

**Review by dsherret — APPROVED** [review]
LGTM

File: cli/tools/lint.rs (hunk 1 above)
Comment (ry) [reply cli/tools/lint.rs:553]: I got a warning locally, but it seems we're not failing on warnings for some reason. I thought this should be doing it: 

https://github.com/denoland/deno/blob/92bb25105822d0def65aa89ee5141c862ac7a3e4/.github/workflows/ci.yml#L150-L153

Comment (dsherret) [reply cli/tools/lint.rs:553]: I think maybe it should be `echo "RUSTFLAGS='-D warnings'" >> $GITHUB_ENV`

Comment (ry) [reply cli/tools/lint.rs:553]: I opened https://github.com/denoland/deno/issues/13998

**Pull request merged**